cluster-autoscaler/cloudprovider/externalgrpc/examples/external-grpc-cloud-provider-service/Makefile

```
@@ -0,0 +1,73 @@
+IMAGE=$(REGISTRY)/ca-external-grpc-cloud-provider$(PROVIDER)
+
+export DOCKER_CLI_EXPERIMENTAL := enabled
+
+build: build-arch-$(GOARCH)
+
+build-arch-%: clean-arch-%
+	$(ENVVAR) GOOS=$(GOOS) GOARCH=$* go build -o ca-external-grpc-cloud-provider-$* ${LDFLAGS_FLAG} ${TAGS_FLAG}
+
+make-image: make-image-arch-$(GOARCH)
+
+make-image-arch-%:
```

checkmake (0.3.2)

[warning] Target "make-image-arch-%" should be declared PHONY.

(phonydeclared)